opensource/project/software-req-win.md

```diff
@@ -65,6 +97,43 @@ To test it, stop the system Docker daemon and start the one you just built:
 
 The other make targets work too, to run unit tests try: `docker run --rm docker-builder sh -c 'cd /c/go/src/github.com/moby/moby; hack/make.sh test-unit'`.
 
+### 6. (Optional) Remove the interim container holding the built executable binaries:
+
+    docker rm binaries
+    
+### 7. (Optional) Remove the image used for the container in which the executable binaries are build. 
+
+Tip - it may be useful to keep this image around if you need to build multiple times. Then you can take advantage of the builder cache to have an image which has all the components required to build the binaries already installed.
+
+    docker rmi nativebuildimage    
+    
+## Validation
+
+The validation tests can only run directly on the host. This is because they calculate information from the git repo, but the .git directory is not passed into the image as it is excluded via .dockerignore. Run the following from a Windows PowerShell prompt (elevation is not required): (Note Go must be installed to run these tests)
+
+   hack\make.ps1 -DCO -PkgImports -GoFormat
+   
+## Unit Tests
+
+To run unit tests, ensure you have created the nativebuildimage above. Then run one ofthe following from an (elevated) Windows PowerShell prompt:
+
+    docker run --rm nativebuildimage hack\make.ps1 -TestUnit
+    docker run --rm -m 2GB nativebuildimage hack\make.ps1 -TestUnit    # (if using Hyper-V containers)
+
+To run unit tests and binary build, ensure you have created the nativebuildimage above. Then run one of the following from an (elevated) Windows PowerShell prompt:
+
+    docker run nativebuildimage hack\make.ps1 -All
+    docker run -m 2GB nativebuildimage hack\make.ps1 -All    # (if using Hyper-V containers)
+
+## Important notes
```

Comment:
I don't think this is a great heading title. Should it be  called something like `Limitations`?